legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-bigquery/src/main/java/org/finos/legend/engine/persistence/components/relational/bigquery/optmizer/StringCaseOptimizer.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.persistence.components.relational.bigquery.optmizer;
+
+import org.finos.legend.engine.persistence.components.physicalplan.PhysicalPlanNode;
+
+import java.util.function.Function;
+
+public abstract class StringCaseOptimizer extends org.finos.legend.engine.persistence.components.relational.ansi.optimizer.StringCaseOptimizer
```

Comment:
Why duplicating this class from the one on the ANSI module?

Reply:
Thanks. Removed, Will resolve the comment